apps/web/src/components/utils/hooks/useMessagePreviews.tsx

```diff
@@ -223,13 +199,10 @@ const useMessagePreviews = () => {
   useEffect(() => {
     const partitionedProfiles = Array.from(messageProfiles).reduce(
       (result, [key, profile]) => {
-        const message = previewMessages.get(key);
-        if (message) {
-          if (profile.isFollowedByMe) {
-            result[0].set(key, profile);
-          } else {
-            result[1].set(key, profile);
-          }
+        if (profile.isFollowedByMe) {
+          result[0].set(key, profile);
+        } else {
+          result[1].set(key, profile);
```

Comment:
Do you think something like this sounds good?

```tsx
result[profile.isFollowedByMe ? 0 : 1].set(key, profile);
```

Reply:
It's a bit harder to read to me, but I'm certainly not opposed to the change.

Reply:
@saagu335 I think @rygine is correct 🙇🏼